reproducibility-with-guix.org

```diff
@@ -36,7 +38,7 @@ guix environment --container --ad-hoc gcc-toolchain
 
 The option =--container= ensures the best possible isolation from the standard environment that your system installation and user account provide for day-to-day work. This environment contains nothing but a C compiler and a shell (which you need to type in commands), and has access to no other files than those in the current directory.
 
-Side note: the option =--container= requires support from the Linux kernel that is not available on all systems. If it doesn't work for you, use =--pure= instead. It provides a less isolated environment, but it is usually more than good enough.
+Side note: the option =--container= requires support from the Linux kernel that is not available on all systems. By default, it is not turned on for unprivileged users so as =root= you need to run =echo 1 > /proc/sys/kernel/unprivileged_userns_clone=. If it doesn't work for you, use =--pure= instead. It provides a less isolated environment, but it is usually more than good enough.
```

Comment:
I didn't do anything on my Guix System to have access to containers as an ordinary user. Is that perhaps specific to a Guix System?

Reply:
I am using Debian stable. So the sentence could be adapted.

Maybe, it is better to place that as a footnote.